rpcserver.go

```diff
@@ -2088,33 +2084,26 @@ func handleGetBlock(s *rpcServer, cmd interface{},
 		return hex.EncodeToString(blkBytes), nil
 	}
 
-	// The verbose flag is set, so generate the JSON object and return it.
-	blockInMainChain, _ := s.chain.MainChainHasBlock(hash)
-	var idx int64
-	if !blockInMainChain {
-		idx = -1
-	}
+	best := s.chain.BestSnapshot()
+
+	// See if this block is an orphan and adjust Confirmations accordingly.
+	onMainChain, _ := s.chain.MainChainHasBlock(hash)
```

Comment:
I think it could but I left the logic as much intact as it was.  There is an identical instance of this lower in this diff.